test/item_repository_test.rb

```diff
@@ -48,50 +48,60 @@ def test_random
 
   def test_find_by_item_id
     item = item_repository.find_by_item_id(7)
+
     assert_equal 7, item.id
   end
 
   def test_find_by_name
-    item = item_repository.find_by_name('Item')
-    assert_equal 'Item', item.name
+    item = item_repository.find_by_name("Item")
+
+    assert_equal "Item", item.name
   end
 
   def test_find_by_item_description
-    item = item_repository.find_by_item_description('Autem Minima,Cumque consequuntur ad.')
-    assert_equal 'Autem Minima,Cumque consequuntur ad.', item.description
+    item = item_repository
+      .find_by_item_description("Autem Minima,Cumque consequuntur ad.")
+
+    assert_equal "Autem Minima,Cumque consequuntur ad.", item.description
   end
 
   def test_find_by_unit_price
+    item = item_repository.find_by_unit_price(BigDecimal.new("31163")/100)
 
-    item = item_repository.find_by_unit_price(BigDecimal.new('31163')/100)
-    assert_equal BigDecimal.new('31163')/100, item.unit_price
+    assert_equal BigDecimal.new("31163")/100, item.unit_price
   end
 
   def test_find_by_merchant_id
      item = item_repository.find_by_merchant_id(1)
+
      assert_equal 1, item.merchant_id
   end
 
   def test_find_all_by_name
-    item = item_repository.find_all_by_name('Item')
+    item = item_repository.find_all_by_name("Item")
+
     assert_equal 3, item.size
   end
 
   def test_find_all_by_description
-    items = item_repository.find_all_by_description('Ea Voluptatum,Sunt officia eum qui molestiae.')
-    items1 = item_repository.find_all_by_description('Autem Minima,Cumque consequuntur ad.')
-    #puts "these are items #{items[0].description }"
+    items = item_repository
+      .find_all_by_description("Ea Voluptatum,Sunt officia eum qui molestiae.")
+    items1 = item_repository
+      .find_all_by_description("Autem Minima,Cumque consequuntur ad.")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.